pkg/apis/storage/types.go

```diff
@@ -158,7 +158,7 @@ type VolumeAttachmentSource struct {
 	// a persistent volume defined by a pod's inline VolumeSource. This field
 	// is populated only for the CSIMigration feature. It contains
 	// translated fields from a pod's inline VolumeSource to a
-	// PersistentVolumeSpec. This field is alpha-level and is only
+	// PersistentVolumeSpec. This field is beta-level and is only
```

Comment:
The versioned types.go (v1beta, v1) also need to be updated

Reply:
Done. I have always been curious what's the relationship between staging/apis and pkg/apis

Reply:
Okay, it is all addressed in the doc: https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api_changes.md#generate-code